Elastiflix/docker-compose-elastic-otel.yml

```diff
@@ -124,12 +125,13 @@ services:
       - OTEL_METRICS_EXPORTER=otlp
       - OTEL_LOGS_EXPORTER=otlp
       - OTEL_TRACES_EXPORTER=otlp
+      - OTEL_JAVAAGENT_DEBUG=${OTEL_JAVAAGENT_DEBUG:-false}
       - REDIS_HOST=redis
       - TOGGLE_SERVICE_DELAY=${TOGGLE_SERVICE_DELAY:-0}
       - TOGGLE_CANARY_DELAY=${TOGGLE_CANARY_DELAY:-0}
       - TOGGLE_CANARY_FAILURE=${TOGGLE_CANARY_FAILURE:-0}
       # agent distribution, either 'otel' or 'elastic'
-      - AGENT_DISTRIBUTION=otel
+      - AGENT_DISTRIBUTION=${AGENT_DISTRIBUTION:-otel}
```

Comment:
[for reviewer] this allows to easily override those environment variables from the `.env`